### ISSUE TITLE
Update wpcs to 0.13.1

### ISSUE THREAD

**Issue by connerbw** [humanmade/coding-standards#23]
"wp-coding-standards/wpcs": "^0.10.0"
https://github.com/humanmade/coding-standards/blob/master/composer.json#L7

Should be 0.13.1:
https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/releases

### Why?

Getting a lot of bogus errors when running Travis CI with PHP 7.2 (nightly)

```
Deprecated: Function create_function() is deprecated in /home/travis/build/[secure]/[secure]/vendor/wp-coding-standards/wpcs/WordPress/Sniffs/WP/PreparedSQLSniff.php 

Warning: count(): Parameter must be an array or an object that implements Countable in /home/travis/build/[secure]/[secure]/vendor/wp-coding-standards/wpcs/WordPress/Sniffs/WP/I18nSniff.php on line 234

...
````

**Comment by connerbw** [comment]
Related:

https://github.com/humanmade/coding-standards/pull/22

**Comment by connerbw** [comment]
PHP 7.2 is planned to be released on November 30...

https://www.youtube.com/watch?v=f4zyjLyBp64

**Comment by rmccue** [comment]
Sorted.
